packages/ripple-address-codec/package.json

```diff
@@ -11,7 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "@scure/base": "^1.1.3",
-    "@xrplf/isomorphic": "^1.0.0"
+    "@xrplf/isomorphic": "^1.0.1"
```

Comment:
It looks like these packages weren't using the latest version of isomorphic. So this is now fixed.